updates: add 2026-04-24 (#79)
dynamical-org/dynamical.org: content/updates/2026-04-24.md

---
title: "Time 2 Chunk (2.0), DWD ICON-EU, status for weather"
date: 2026-04-24
---

Listen, my second child is arriving any minute now. This transmission will be brief. I haven't decided if I will name him Poseidon, Gefs, or Turbulence.

> ## Zarr v2 deprecation

Reviewer: ```suggestion
## Zarr v3 deprecation
```

We're all-in on Icechunk. Our existing Zarr v2 datasets will continue updating for 90 days, at which point we will sunset them. Consider them deprecated and **update by July 23, 2026**.

## Speaking of which

The new access pattern:

### Via dynamical-catalog

```python
import dynamical_catalog

# Open a dataset as an xarray Dataset via its Icechunk repository
ds = dynamical_catalog.open("noaa-gfs-forecast")

# List all available datasets

> dynamical_catalog.list()

Reviewer: I'd move this down into the 2nd example code block

```

To more easily ensure the right set of dependencies and to make an increasingly variable set of opening incantations more accessible, we made a python library to list and open datasets.

```python
import dynamical_catalog

# Get the underlying Zarr store if you want even more control
store = dynamical_catalog.get_store("noaa-gfs-forecast")
ds = xr.open_zarr(store)
```

### Via STAC

> We are STAC-maxxers now. https://stac.dynamical.org/catalog.json hosts ALL of the data used to create our documentation and catalogs. Point your robots at it for all the documentation they need to get going. STAC-as-skill.

Reviewer: ```suggestion
We are STAC-maxxers now. https://stac.dynamical.org/catalog.json contains ALL of the data used to create our documentation and drive the dynamical-catalog library. Point your robots at it for all the documentation they need to get going. STAC-as-skill.
```

## ICON-EU

Deutscher Wetterdienst's [ICON-EU](https://dynamical.org/catalog/dwd-icon-eu-forecast-5-day/) is now a cloud-optimized, live-updating Icechunk 2.0 Zarr. High-resolution 5-day forecasts over Europe, nested from DWD's global ICON model, at the 00/06/12/18 UTC cycles.

What makes this one extra fun: as far as we can tell, **no other public archive of ICON-EU exists**. DWD only keeps a short rolling window on their own servers, so we'll keep growing the archive from here.

Huge thanks to:

> - [Jack Kelly](https://www.linkedin.com/in/jackkelly0/) and [Open Climate Fix](https://openclimatefix.org/) for getting this off the ground and hoarding forecasts to kickstart the archive

Reviewer: ```suggestion
- [Jack Kelly](https://www.linkedin.com/in/jackkelly0/) from [Open Climate Fix](https://openclimatefix.org/) for getting this off the ground and hoarding forecasts to kickstart the archive and to the [Patrick J McGovern Foundation](https://www.mcgovern.org/) for supporting their work.
```
